pkg/apis/config/default.go

```diff
@@ -78,27 +76,11 @@ func NewDefaultsFromMap(cfgMap map[string]string) (*Defaults, error) {
 	}
 
 	if defaultRunAsUser, ok := cfgMap[defaultRunAsUserKey]; ok {
```

Comment:
Shouldn't this be 
```suggestion
	if defaultRunAsUser, ok := cfgMap[defaultRunAsUserKey]; !ok {
```
now? Otherwise, we are always setting the default value now.

Reply:
:thinking: 
```
	if defaultRunAsUser, ok := cfgMap[defaultRunAsUserKey]; ok {
		tc.DefaultRunAsUser = defaultRunAsUser
	}
```
overriding tc.DefaultRunAsUser to whatever coming from CM

Reply:
Sorry. Read that line incorrectly.